appdatabase/database.go

```diff
@@ -76,6 +85,37 @@ func InitializeDB(path, password string, kdfIterationsNumber int) (*sql.DB, erro
 	return db, nil
 }
 
+func fixMissingKeyUIDForAccounts(sqlTx *sql.Tx) error {
+	rows, err := sqlTx.Query(`SELECT address,pubkey FROM accounts WHERE pubkey IS NOT NULL`)
```

Comment:
@qfrank well, this is ok, in terms that the `keyUid` is an account's `keyUid`, but this field should refer to the key pair's `keyUid`, which is a `keyUid` created from the `PublicKey` of the derivation path `m` of the seed phrase this account was derived from. We use that field to know which accounts belong to the same key pair.

Please re-read [this comment](https://github.com/status-im/status-go/pull/5203#discussion_r1611593491) and pay attention to the third point from the "Solution" part, that's for discussion and if we want to deal with it properly we have to follow one of the proposals given in "Decision for above proposed solution:" part of that comment.